ci/environment-2.7.yml

```diff
@@ -40,3 +41,4 @@ dependencies:
     - git+https://github.com/dask/distributed.git
     - git+https://github.com/dask/dask-tensorflow.git
     - git+https://github.com/dask/dask-xgboost.git
+    - graphviz
```

Comment:
FWIW there is a conda-forge package call `python-graphviz`, which is equivalent to this and requires `graphviz` (the system one).

Reply:
Awesome, good to know! I'll check `python-graphviz` out, thanks for the tip